config/hooks/post-merge

```diff
@@ -10,9 +10,9 @@ trap "rm -f ${files_changed_upstream}" EXIT
 
 git diff-tree -r --name-only --no-commit-id ORIG_HEAD HEAD > "$files_changed_upstream"
 
-grep -E --quiet 'requirements-(base|dev)\.txt' "$files_changed_upstream" && py="install-py-dev "
-grep -E --quiet 'yarn\.lock' "$files_changed_upstream"                   && js="install-js-dev "
-grep -E --quiet 'migrations' "$files_changed_upstream"                   && migrations="apply-migrations "
+grep --quiet 'requirements-dev-frozen.txt' "$files_changed_upstream" && py="install-py-dev "
```

Comment:
We don't want to track base anymore?

Reply:
No, since `requirements-dev-frozen.txt` is what people's venvs will install from: `extras_require["dev"] = get_requirements("dev-frozen")`

If like, base changes and dev-frozen doesn't reflect that, the new lint-requirements will catch that and it will fail. So dev-frozen is all we need to look at.